src/hooks/useVenueUserStatuses.ts

```diff
@@ -0,0 +1,32 @@
+import { DefaultUserStatus, User } from "types/User";
+
+import { USER_STATUSES } from "settings";
+import { WithId } from "utils/id";
+import { useRelatedVenues } from "./useRelatedVenues";
+import { useSovereignVenueId } from "./useSovereignVenueId";
+
+export const useVenueUserStatuses = (venueId?: string, user?: WithId<User>) => {
+  const { sovereignVenueId } = useSovereignVenueId({ venueId });
+
+  // Using currentVenueNG hook
```

Comment:
not absolutely sure what the comment means. 🙈 Can you explain, please?

Reply:
I left this note to remind myself about this issue. We need to build useSovereignVenue hook and use it instead of this. Otherwise these is no substitute.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/71f5181e9d01672881376d7c0a0f0f0f917511eb

Reply:
Do we not already have that hook?